deploy/rbac-permissions-operator-config/05-dedicated-admins-aggregate-cluster.ClusterRole.yaml

```diff
@@ -316,3 +316,22 @@ rules:
   - list
   - watch
 ### END
+### Start - allow dedicated admin to create consoleplugin resource and enable/disable console plugin in the cluster
+# https://issues.redhat.com/browse/OSD-13617
+- apiGroups:
+  - console.openshift.io
+  resources:
+  - consoleplugins
+  verbs:
```

Comment:
Was the read permission (like get, list) already given in some other places? I think it would make sense to grant them together?

Reply:
Yes already given [here](https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml#L131-L140). That is very generic for all the resources under `console.openshift.io` group.